airbyte-integrations/bases/connector-acceptance-test/unit_tests/data/docs/correct_all_description_exist.md

```diff
@@ -0,0 +1,38 @@
+# GitHub
+
+<HideInUI>
+
+This page contains the setup guide and reference information for the [GitHub](https://www.github.com) source connector.
+
+</HideInUI>
+
+## For Airbyte Cloud:
+
+1. [Log into your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
+2. Click Sources and then click + New source/destination.
+3. On the Set up the source page, select GitHub from the Source type dropdown.
+4. Enter a name for the GitHub connector.
+5. Add list of GitHub repositories you want to sync.
+6. Add Start Date from with data will be replicated.
+
+## For Airbyte Open Source:
+
+1. Navigate to the Airbyte Open Source dashboard.
+2. Click Sources and then click + New source/destination.
+3. On the Set up the source page, select GitHub from the Source type dropdown.
+4. Enter a name for the GitHub connector.
+5. Add list of GitHub repositories you want to sync.
+6. Add Start Date from with data will be replicated.
+
+## Supported sync modes
+
+The GitHub source connector supports the following [sync modes](https://docs.airbyte.com/cloud/core-concepts/#connection-sync-modes):
+
+- Full Refresh 
+- Incremental
+
+## Tutorials
+
+Now that you have set up the GitHub source connector, check out the following GitHub tutorials:
+
+- [Creating PAT](https:github.com/how-to-create-a-pat)
```

Comment:
Do we need another check for valid URL links? At least assertion for `status_code`

Reply:
added